make-util.js

```diff
@@ -392,6 +392,9 @@ var downloadFile = function (url) {
 }
 exports.downloadFile = downloadFile;
 
+// string hashing from https://stackoverflow.com/questions/7616461/generate-a-hash-from-string-in-javascript
+const hashCode = s => s.split('').reduce((a,b)=>{a=((a<<5)-a)+b.charCodeAt(0);return a&a},0);
```

Comment:
can you add a comment why this is necessary for those who don't have context

Reply:
removed